package.json

```diff
@@ -19,13 +19,28 @@
   "license": "ISC",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-sample.git"
+    "url": "git+https://github.com/blackbaud/skyux-list-builder-view-grids.git"
   },
   "bugs": {
-    "url": "https://github.com/blackbaud/skyux-sample/issues"
+    "url": "https://github.com/blackbaud/skyux-list-builder-view-grids/issues"
+  },
+  "homepage": "https://github.com/blackbaud/skyux-list-builder-view-grids#readme",
+  "peerDependencies": {
+    "@angular/common": "^4.3.6",
+    "@angular/core": "^4.3.6",
+    "@skyux/animations": ">=3.0.0-rc.0",
+    "@skyux/forms": ">=3.0.0-rc.0",
+    "@skyux/grids": "3.0.0-alpha.1",
+    "@skyux/indicators": ">=3.0.0-rc.1",
+    "@skyux/layout": ">=3.0.0-rc.0",
+    "@skyux/list-builder": "3.0.0-alpha.0",
+    "@skyux/lists": ">=3.0.0-rc.0",
+    "@skyux/lookup": ">=3.0.0-rc.0",
+    "@skyux/modals": ">=3.0.0-rc.0",
+    "@skyux/popovers": ">=3.0.0-rc.0",
```

Comment:
I only saw imports from the following SKY UX modules. Should the other ones be removed?
```
"@skyux/grids": "^3.0.0-alpha.1",
"@skyux/indicators": "^3.0.0-rc.1",
"@skyux/list-builder": "^3.0.0-alpha.0"
```

Reply:
Without them, we have unmet peer dependencies from list-builder, and the app won't serve properly. I can always install them as dev-dependencies to get the local serve to work, but leave them out of peer dependencies and rely on the consumer to make sure they're installed. Whats the right move here?

Reply:
I would put the missing modules (the ones your library does not directly import) in the dev dependencies.

Reply:
done.